### ISSUE TITLE
[Bug]: Crash on installing apk on miui

### ISSUE THREAD

**Issue by MohammedKHC** [AndroidIDEOfficial/AndroidIDE#882]
### What happened?

When the build finish the app crashed

### What's the expected behavior?

Not to crash

### What version of AndroidIDE you're using?

v2.4.0

### Relevant log output

```shell
AndroidIDE crash report
Manufacturer: Xiaomi
Device: M2010J19SG
ABI: arm64-v8a
SDK version: 31
App version: v2.4.1-beta (241)

 Stacktrace: 
java.lang.IllegalArgumentException: Failed to find configured root that contains /data/data/com.itsaky.androidide/files/home/Create Block/app/build/outputs/apk/debug/app-debug.apk
	at androidx.core.content.FileProvider$SimplePathStrategy.getUriForFile(FileProvider.java:175)
	at com.itsaky.androidide.utils.ApkInstaller.installApk(ApkInstaller.kt:146)
	at com.itsaky.androidide.actions.build.QuickRunAction$$ExternalSyntheticLambda0.run(R8$$SyntheticClass:77)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:211)
	at android.os.Looper.loop(Looper.java:300)
	at android.app.ActivityThread.main(ActivityThread.java:8322)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:556)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1038)
```

### Duplicate issues

- [X] This issue has not been reported yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by itsaky** [comment]
I tried running a project located in the `/data/data/com.itsaky.androidide/files/home` directory, but I'm unable to reproduce this issue. Could you provide instructions for the same? Or probably a sample project?

**Comment by MohammedKHC** [comment]
Actually i tried using the default AndroidX template and its causing crash, Anyways i made an pull request to fix the bug please review it #888
